config/config.go

```diff
@@ -564,6 +566,9 @@ func (c *Config) Valid() error {
 	if c.TiKVClient.MaxTxnTimeUse == 0 {
 		return fmt.Errorf("max-txn-time-use should be greater than 0")
 	}
+	if c.Performance.AutoIDStep <= 0 {
```

Comment:
I think we should prevent the `step` is too big, what about setting 50w as the max `step`?